nixos/tests/hedgedoc.nix

```diff
@@ -33,10 +33,19 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
         };
         postgresql = {
           enable = true;
-          initialScript = pkgs.writeText "pg-init-script.sql" ''
-            CREATE ROLE hedgedoc LOGIN PASSWORD 'snakeoilpassword';
-            CREATE DATABASE hedgedocdb OWNER hedgedoc;
-          '';
+          ensureDatabases = [
+            {
+              name = "hedgedocdb";
+              owner = "hedgedoc";
+            }
+          ];
+          ensureUsers = [
+            {
+              name = "hedgedoc";
+              passwordFile = pkgs.writeText "hedgedoc-password" "snakeoilpassword";
+              ensureClauses.login = true;
+            }
+          ];
```

Comment:
I don't know why the dbname and owner don't match here and why not socket auth is used but I also don't have enough time to also maintain the tests of my packages. sigh. :(